packages/s3/.gitignore

```diff
@@ -1 +1,2 @@
 test/results
+coverage/
```

Comment:
Already ignored in top `.gitignore`. Is that not covering it?

Reply:
I tend to be explicit in each package, so that if we edit the one at the root, we are sure to not send this directory by mistake.

I can remove it.

Reply:
No problem, let's keep it.
No other package needed that anyway, because we're only collecting coverage from workspace root and there are no explicit `coverage` scripts anywhere